src/javascript/_common/base/login.js

```diff
@@ -11,7 +10,7 @@ const getAppId            = require('../../config').getAppId;
 
 const Login = (() => {
     const redirectToLogin = () => {
-        if (!Client.isLoggedIn() && !isLoginPages() && isStorageSupported(sessionStorage)) {
+        if (!isLoginPages() && isStorageSupported(sessionStorage)) {
```

Comment:
why?

Reply:
I think we need this here because this function is used is many places and not just reset password. if we want to ignore it in some cases, we can pass a flag to the function.

Reply:
Since we do not need authentication for this page, we can revert back this change.